feature-libs/organization/account-summary/core/account-summary-page-meta.resolver.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
+import {
+  BreadcrumbMeta,
+  ContentPageMetaResolver,
+  Page,
+  RoutingService,
+  SemanticPathService,
+  TranslationService,
+} from '@spartacus/core';
+import { OrganizationPageMetaResolver } from '@spartacus/organization/administration/core';
+import { combineLatest, defer, Observable } from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+  shareReplay,
+  switchMap,
+} from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AccountSummaryPageMetaResolver extends OrganizationPageMetaResolver {
+  constructor(
+    protected contentPageMetaResolver: ContentPageMetaResolver,
+    protected translation: TranslationService,
+    protected semanticPath: SemanticPathService,
+    protected routingService: RoutingService
+  ) {
+    super(contentPageMetaResolver, translation, semanticPath, routingService);
+  }
+
+  route = 'orgAccountSummary';
```

Comment:
can we do something like this :`protected readonly ORGANIZATION_TRANSLATION_KEY = 'organization.breadcrumb';`

Reply:
done